jmix-dynattr/dynattr-flowui/src/main/java/io/jmix/dynattrflowui/view/categoryattr/CategoryAttributesDetailView.java

```diff
@@ -684,6 +693,10 @@ protected void refreshAttributesUI() {
             component.setVisible(visible);
 
             if (!visible && component instanceof HasValue) {
+                if (component.equals(defaultStringField) && ENUMERATION.equals(attributeType) &&
```

Comment:
If we continueing if condition from new line, lets split each condition from new line like this:
(cond1 &&
cond2 &&
cond3)